src/Toolkit/Date.php

```diff
@@ -70,20 +69,17 @@ public function ceil(string $unit)
 	/**
 	 * Returns the interval between the provided and the object's datetime
 	 *
-	 * @param string|int|\DateTimeInterface $datetime
-	 * @param \DateTimeZone|null $timezone Optional default timezone if `$datetime` is string
-	 * @return \DateInterval
+	 * @param \DateTimeZone $timezone Optional default timezone if `$datetime` is string
 	 */
-	public function compare($datetime = 'now', ?DateTimeZone $timezone = null)
-	{
+	public function compare(
+		string|int|DateTimeInterface $datetime = 'now',
+		DateTimeZone $timezone = null
+	): DateInterval {
```

Comment:
Should we consider only this PR or the whole Toolkit package? For example, this change is not applied to `Date::__construct()`. Do you want me to complete the missing ones for the whole Toolkit package with a new PR?

Reply:
Let's start with these classes first and open the next PR afterwards. Otherwise it's going to be too much at once. :)

Reply:
Let's start with these classes first and open the next PR afterwards. Otherwise it's going to be too much at once. :)

Reply:
OK, you're right 👍